### DISCUSSION TITLE
Unable connection migration！

### DISCUSSION THREAD

**Discussion by msszzj** [microsoft/msquic#2139] [Q&A] [answered]
Hi, I'm using "sample. c" to test connection migration.
I have two Network Interface Card. However, after I close one, quic connection cannot be migrated to another card.
As a client, it seems that some settings are missing. Do I need to set any parameters for the connection? 
Is it because the sourcecid is 0 that the migration cannot be performed.
thank you!

**Comment by nibanks** [comment, marked as answer]
Interesting scenario, but unfortunately that would currently fall under:
- https://github.com/microsoft/msquic/issues/1575

Which is not yet complete. We do support the case where a client is implicitly migrated via NAT rebinding. But for this kind of scenario, we'd currently require you to explicitly change the interface you're using from one to the other, on the client side. And that requires some work still.

**Reply by msszzj** [reply]
Thank you very much. 
I tried to set a new address for the connection through "setparam()", but I also returned an error.
When I found that the Network Interface Card was closed,  using ”setparam()“ to update the IP address is not supported at present.

**Reply by nibanks** [reply]
Yeah, using `SetParam` will be the correct way to do it, there's just a significant amount of work to wire everything up. We welcome outside contributions if you're interested! 😆

**Reply by msszzj** [reply]
Thank you!
